src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala

```diff
@@ -323,20 +248,22 @@ object HouseholdActor {
     override def receive: Receive = {
 
       case NotifyVehicleResourceIdle(
-          vehId: Id[BeamVehicle],
```

Comment:
Why the change here? An exhaustive warning? If so then my question is that this just hides the warning. Is there a scenario where the cast would fail?

Reply:
Same for all the other areas of the same type below